src/harness/testcases/WINNF_FT_S_FPR_testcase.py

```diff
@@ -0,0 +1,154 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import sas
+import sas_testcase
+import logging
+from util import winnforum_testcase, writeConfig, loadConfig, configurable_testcase,\
+ addCbsdIdsToRequests
+
+class FSSProtectionTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+    
+  def generate_FPR_5_default_config(self, filename):
+    """Generates the WinnForum configuration for FPR.5."""
+
+    # Load FSS station operating at range 3600 - 3700 MHz.
+    fss = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
```

Comment:
Please refer the changes to fss_record_0.json in PR#486. 

So it needs to be:
fss_data = json.load(open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
fss =  fss_data['record']